uflash.py

```diff
@@ -69,11 +71,13 @@ def hexlify(script):
     # Convert line endings in case the file was created on Windows.
     script = script.replace(b'\r\n', b'\n')
     script = script.replace(b'\r', b'\n')
+    if len(script) > _MAX_SIZE:
+        script = nudatus.mangle(script)
```

Comment:
Looks like nudatus already contains the micro:bit internal flash size, but it could be used as a generic tool for any MicroPython code, so perhaps the size could be passed as an another argument?
This would also be helpful to use for `main.py` file flashed via filesystem, as it can be larger than 8KBs, but still has a limit.

Reply:
I'm not aware of nudatus referencing the micro:bit flash size as I did intend for it to be independent, this len comparison is just to prevent unnecessary work

Makes sense as for nudatus to be an optional dependency though

Reply:
Sorry, you are right, not quite sure why I got that impression. It's already generic as it is.

Reply:
Don't worry, your probably remembering the original PR in Mu